src/toil_scripts/exome_variant_pipeline/exome_variant_pipeline.py

```diff
@@ -607,25 +613,39 @@ def generate_config():
     # URLs can take the form: http://, file://, s3://, gnos://.
     # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank
     ####################################################################################################################
-    reference:              # Required: URL to reference genome
-                            # Example: s3://cgl-pipeline-inputs/variant_hg19/hg19.fa\n
-    phase:                  # Required: URL to phase indels VCF
-                            # Example: s3://cgl-pipeline-inputs/variant_hg19/1000G_phase1.indels.hg19.sites.vcf\n
-    mills:                  # Required: URL to Mills indel VCF
-                            # Example: s3://cgl-pipeline-inputs/variant_hg19/Mills_and_1000G_gold_standard.indels.hg19.sites.vcf\n
-    dbsnp:                  # Required: URL to dbsnp VCF
-                            # Example: s3://cgl-pipeline-inputs/variant_hg19/dbsnp_138.hg19.vcf\n
-    cosmic:                 # Required: URL to cosmic VCF
-                            # Example: s3://cgl-pipeline-inputs/variant_hg19/cosmic.hg19.vcf\n\n
-    run-mutect: true        # Optional: If true, will run MuTect to do mutation calls
-    run-pindel: true        # Optional: Iff true, will run pindel to analyze indels
-    run-muse: true          # Optional: If true, will run MuSe to do mutation calls
-    preprocessing: true     # Optional: If true, will perform indel realignment and base quality score recalibration\n
-    output-dir:             # Optional: Provide a full path to where results will appear\n
-    s3-dir:                 # Optional: Provide an s3 path (s3://bucket/dir) where results will appear\n
-    ssec:                   # Optional: Provide a full path to a 32-byte key used for SSE-C Encryption in Amazon\n
-    gtkey:                  # Optional: Provide a full path to a CGHub Key used to access GNOS hosted data\n
-    ci-test:                # Optional: If true, uses resource requirements appropriate for continuous integration\n
+    # Required: URL to reference genome
+    # Example: s3://cgl-pipeline-inputs/variant_hg19/hg19.fa
+    reference: \n
```

Comment:
And I would get rid of the `\n` to generate empty lines in the output without having them in the code. Either put empty lines in the code or remove them altogether.